halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/local/git/LocalGitDeckService.java

```diff
@@ -57,17 +58,26 @@
 
   @Override
   public ServiceSettings buildServiceSettings(DeploymentConfiguration deploymentConfiguration) {
-    boolean authEnabled = deploymentConfiguration.getSecurity().getAuthn().isEnabled();
-    UiSecurity uiSecurity = deploymentConfiguration.getSecurity().getUiSecurity();
-    if (uiSecurity.getSsl().isEnabled()) {
-      setStartCommand(String.join("\n","export DECK_HTTPS=true", getStartCommand()));
+    Security security = deploymentConfiguration.getSecurity();
+    if (security.getUiSecurity().getSsl().isEnabled()) {
+      setEnv("DECK_HTTPS");
     }
-    return new Settings(uiSecurity)
+    if (security.getAuthn().isEnabled()) {
+      setEnv("AUTH_ENABLED");
+    }
+    if (security.getAuthz().isEnabled()) {
+      setEnv("FIAT_ENABLED");
+    }
+    return new Settings(security.getUiSecurity())
         .setArtifactId(getArtifactId(deploymentConfiguration.getName()))
-        .setHost(authEnabled ? "0.0.0.0" : getDefaultHost())
+        .setHost(security.getAuthn().isEnabled() ? "0.0.0.0" : getDefaultHost())
         .setEnabled(true);
   }
 
+  private void setEnv(String var) {
+    setStartCommand(String.join("\n", "export " + var + "=true", getStartCommand()));
```

Comment:
Genuine question - do we need to `export` the var? [This SO post](https://stackoverflow.com/a/10856348/5569046) appears to use some bash magic that I've seen @ewiseblatt use with the python scripts.